### PR TITLE
OHRM5X-1245: System Check Modifications

### PR DESCRIPTION
**Pull request by SandeepaOHRM**
This PR contains modifications regarding the review feedback of following PR
https://github.com/orangehrm/orangehrm/pull/1227

### CODE REVIEW

File: installer/Util/Messages.php [hunk 1]
```diff
@@ -21,56 +21,42 @@
...
 
-    public const WritableSymfonyCache_OK_MESSAGE = "Write Permissions for src/cache - Writeable";
-    public const WritableSymfonyCache_FAIL_MESSAGE = "Write Permissions for src/cache - Not Writeable";
+    public const WRITEABLE_SYMFONY_CONFIG_OK_MESSAGE = "Write Permissions for - src/config - Writeable";
+    public const WRITEABLE_SYMFONY_CONFIG_FAIL_MESSAGE = "Write Permissions for - src/config - Not Writeable";
 
-    public const WritableSymfonyLog_OK_MESSAGE = "Write Permissions for src/log - Writeable";
-    public const WritableSymfonyLog_FAIL_MESSAGE = "Write Permissions for src/log - Not Writeable";
+    public const WRITEABLE_SYMFONY_CACHE_OK_MESSAGE = "Write Permissions for src/cache - Writeable";
+    public const WRITEABLE_SYMFONY_CACHE_FAIL_MESSAGE = "Write Permissions for src/cache - Not Writeable";
```
Comment (RajithaKumara) [inline installer/Util/Messages.php:42]: `SYMFONY` changed to `SRC`

File: installer/Util/Messages.php [hunk 2]
```diff
@@ -21,56 +21,42 @@
...
 
-    public const MaximumSessionIdle_OK_MESSAGE = "Maximum Session Idle Time before Timeout - Good";
-    public const MaximumSessionIdle_SHORT_MESSAGE = "Maximum Session Idle Time before Timeout - Short";
-    public const MaximumSessionIdle_TOO_SHORT_MESSAGE = "Maximum Session Idle Time before Timeout - Too short";
+    public const WRITEABLE_SYMFONY_LOG_OK_MESSAGE = "Write Permissions for src/log - Writeable";
+    public const WRITEABLE_SYMFONY_LOG_FAIL_MESSAGE = "Write Permissions for src/log - Not Writeable";
```
Comment (RajithaKumara) [inline installer/Util/Messages.php:45]:
```suggestion
    public const WRITEABLE_SRC_LOG_FAIL_MESSAGE = "Write Permissions for src/log - Not Writeable";
```
refactor rename

File: installer/Util/SystemConfig.php [hunk 3]
```diff
@@ -48,10 +45,16 @@ class SystemConfig
...
+    private ?UpgraderConfigUtility $upgraderConfigUtility = null;
+    private $systemRequirements;
 
     public function __construct()
     {
         $this->filesystem = new Filesystem();
+        $this->upgraderConfigUtility = new UpgraderConfigUtility();
+        $this->systemRequirements = Yaml::parse(
+            file_get_contents(realpath(__DIR__ . '/../environmentCheck/system_requirements.yml'))
+        );
```
Comment (RajithaKumara) [inline installer/Util/SystemConfig.php:57]:
```suggestion
        $this->systemRequirements = Yaml::parseFile
            realpath(__DIR__ . '/../environmentCheck/system_requirements.yml')
        );
```

File: installer/Util/SystemConfig.php [hunk 4]
```diff
@@ -48,10 +45,16 @@ class SystemConfig
 
     private bool $interruptContinue = false;
     private ?Filesystem $filesystem = null;
+    private ?UpgraderConfigUtility $upgraderConfigUtility = null;
+    private $systemRequirements;
```
Comment (RajithaKumara) [inline installer/Util/SystemConfig.php:49]:
```suggestion
    private array $systemRequirements;
```

**Review by RajithaKumara — CHANGES_REQUESTED** [review]

File: installer/Util/SystemConfig.php (hunk 3 above)
Comment (SandeepaOHRM) [reply installer/Util/SystemConfig.php:57]: done

File: installer/Util/Messages.php (hunk 1 above)
Comment (SandeepaOHRM) [reply installer/Util/Messages.php:42]: done

File: installer/Util/Messages.php (hunk 2 above)
Comment (SandeepaOHRM) [reply installer/Util/Messages.php:45]: done

**Review by RajithaKumara — APPROVED** [review]

**Pull request merged**